setup.py

```diff
@@ -98,15 +98,20 @@
                     os.path.relpath(root, PACKAGENAME), filename))
 package_info['package_data'][PACKAGENAME].extend(c_files)
 
+setup_requires = metadata.get('requires-dist')
+if setup_requires is None:
+    setup_requires =  ['numpy', 'cython', 'astropy']
+else:
+    # Convert the config setup_requires string into a list
+    setup_requires = setup_requires.strip().split()
 # Note that requires and provides should not be included in the call to
 # ``setup``, since these are now deprecated. See this link for more details:
 # https://groups.google.com/forum/#!topic/astropy-dev/urYO8ckB2uM
-
 setup(name=PACKAGENAME,
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['astropy'],
+      setup_requires=setup_requires,
```

Comment:
The above defaults are meant to stop failures from using the astropy_helpers module which is included in the package template. If any of those 3 packages are not installed, running python setup.py will crash. By adding those packages to setup_requires. If those packages are not installed, distutils will try to automatically install them. I think it might make sense to include these defaults in setup_requires and take the install_requires from the setup.cfg file. What do you think?